.github/workflows/build-scanner-tarball.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: 'Check out, build, pack'
         run: |
           # Check out the target branch.
-          git clone https://github.com/forcedotcom/sfdx-scanner.git sfdx-scanner
+          git clone -b dev-4 https://github.com/forcedotcom/sfdx-scanner.git sfdx-scanner
           cd sfdx-scanner
```

Comment:
Suggestion...

Maybe we call the folder sfdx-scanner-v4 instead of sfdx-scanner to make it clear. I'll let you decide.

Reply:
Let's leave it as is. Anyways the folder is ephemeral and we will revisit this logic soon and ideally this should all go away. I also just started a full run testing, don't want to redo that either.